server/src/categories/entities/category.entity.ts

```diff
@@ -1,17 +1,22 @@
 import { Entity, Column, PrimaryGeneratedColumn, Index } from 'typeorm';
+import { ApiProperty } from '@nestjs/swagger';
 
 @Entity('categories')
 export class Category {
+  @ApiProperty()
   @PrimaryGeneratedColumn()
   id: number;
```

Comment:
You can get the same behavior by removing the annotation and adding the optional property denotator:

```suggestion
  @PrimaryGeneratedColumn()
  id: number?;
```